.github/actions/deploy-ubuntu/action.yml

```diff
@@ -80,8 +81,9 @@ runs:
         $SUDO apt-get -y install gnupg
         source /etc/os-release
         export CODENAME=$UBUNTU_CODENAME
-
-        if [[ ! "$ARCH" == "amd64" ]]; then
+        # arm64 runners don't work with apt-spy2,
+        # the selected mirrors don't have arm64 binaries
+        if [[ ! "$ARCH" == "amd64" && ! "$ARCH" == "arm64" ]]; then
```

Comment:
This needs to be enabled for cross-compilation though, so disable it only when it's on an ARM runner

Reply:
done, replaced the guard with `if [[ "$ARCH" != "arm64" ]]; then`